### ISSUE TITLE
Simulation of example "com.yakindu.sct.examples.trafficlight.multism" does not work

### ISSUE THREAD

**Issue by tkutz** [Yakindu/examples#209]
I think the example itself is not very useful to demonstrate a simulation of multi statemachines. There are at least two issues: (1) the submachines are never entered, (2) all statecharts are cyclebased, but the submachines' runCycle() method is never called

This results in a simulation where nothing happens, which gives the impression that something is broken.

**Comment by rherrmannr** [comment]
is this still an issue @tkutz ?

**Comment by tkutz** [comment]
Yes, simulating TwoWayTrafficControl.ysc does not do much because the sub machines are never entered.
